src/importNameRule.ts

```diff
@@ -84,15 +84,15 @@ class ImportNameRuleWalker extends ErrorTolerantWalker {
     }
 
     private validateImport(node: ts.ImportEqualsDeclaration | ts.ImportDeclaration, importedName: string, moduleName: string): void {
-        moduleName = moduleName.replace(/.*\//, ''); // chop off the path
-        moduleName = this.makeCamelCase(moduleName);
-        if (this.isImportNameValid(importedName, moduleName) === false) {
-            const message: string = `Misnamed import. Import should be named '${moduleName}' but found '${importedName}'`;
+        let expectedImportedName = moduleName.replace(/.*\//, ''); // chop off the path
```

Comment:
I changed the variable name `moduleName` to `expectedModuleName` as it was confusing.<br> Also the primary reason for new variable was to make sure that we aren't changing the input we get which the code was doing earlier.